evalml/utils/woodwork_utils.py

```diff
@@ -101,6 +103,9 @@ def _retain_custom_types_and_initalize_woodwork(
         return ww.init_series(new_dataframe, old_logical_types)
     if ltypes_to_ignore is None:
         ltypes_to_ignore = []
+    old_logical_types = {
+        k: v if inspect.isclass(v) else type(v) for k, v in old_logical_types.items()
```

Comment:
Do need `isclass` here? The types should always be instances from now on right?

Reply:
IIRC some tests were failing without this change but I'll double check this!